### PR TITLE
Create consensus-depin-austin.md

### PR DESCRIPTION
**Pull request by laeccles**
@WinsCodes New Event.  Can you add these host logos?

impossiblecloud_logo
outlierventures_logo

### CODE REVIEW

File: content/events/consensus-depin-austin.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+#Preferred means of contacts - this is a placeholder for email(for eg:  - email|mailto:<email_id>) and other social handles like Twitter, LinkedIn, Discord, etc. (for eg.   - 'twitter|https://twitter.com/IPFS/status/1629199396700098560?s=20')
+
+preferredContacts:
+  - '<contact_means>|<contact_url>'
+
+#Event host names and their respective logos(preferred size is 48px width, 48px height)-place the logo file on the path 'public/uploads' for eg.   - IPFS|ipfs-logo.png
+
+eventHosts:
+  - Impossible Cloud|<host_logo>
```
Comment (WinsCodes) [inline content/events/consensus-depin-austin.md:42]: @laeccles Please share logos for all the hosts and mention their names along with them to identify the same. Cc @pandeyanujk

**Comment by laeccles** [comment]
@WinsCodes 

Impossible cloud
impossiblecloud_logo

peaq
peaq_logo

Borderless Capital
v4wKsUAM_400x400

Outlier Ventures

outlierventures_logo

File: content/events/consensus-depin-austin.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+eventType: "Conference" #Type of the event: Please choose one among the below options or just leave it blank
+
+location: "Austin, United States" #Specify the location of the event.If you aren't sure about the location then mention "Location TBD"
+
+venueName: "The Perch ATX" #the event venue name (will show up on the event card) or just leave it blank
+
+venueAddress: "74 Trinity St, Austin, TX 78701" #the event venue address (will show up on a map) or just leave it blank
+
+venueMapsLink: "https://maps.app.goo.gl/74hSTrnR4oaKdzb39" #the event venue Map link (will show up on a map) or just leave it blank
```
Comment (WinsCodes) [inline content/events/consensus-depin-austin.md:22]: @laeccles This map link does not seem to work. Can yu please check this.

**Comment by WinsCodes** [comment]
Event added using the pull request - https://github.com/memser-spaceport/pln-events/pull/476

**Pull request closed without merging**